lib/tag-list/index.tsx

```diff
@@ -78,7 +82,16 @@ const SortableTag = SortableElement(
         }}
         value={tag.name}
       />
-      {editingActive && allowReordering && <TagHandle />}
+      {editingActive && (
+        <button className="button-borderless">
+          <TrashIcon onClick={() => trashTag(tag.name)} />
+        </button>
+      )}
+      {editingActive && allowReordering && (
+        <button className="theme-color-fg-dim">
```

Comment:
Looks like the trash icon should be `$studio-simplenote-blue-30` in dark mode and `$studio-simplenote-blue-50` in light mode. (I know it makes it look like we should use `button-borderless` for these because they're the same color but the borderless buttons are actually something different, updates to that coming soon, sorry!!)

The drag handles are lighter, which you did notice already -- `$studio-gray-50` in dark mode and `$studio-gray-10` in light mode. I pinged Sly to double-check that those should be different from the other icon colors.

Reply:
Update: Confirmed the handles are intended to be a lighter color than the other gray icons

Reply:
Thanks! I've updated this based on the suggestion and changed them to be icon-button and updated the colors using this in the component.